bootstrap/node_logic.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/orbs-network/orbs-network-go/services/consensusalgo/leanhelixconsensus"
 	"github.com/orbs-network/orbs-network-go/services/consensuscontext"
 	"github.com/orbs-network/orbs-network-go/services/crosschainconnector/ethereum"
+	ethereumConnectorAdapter "github.com/orbs-network/orbs-network-go/services/crosschainconnector/ethereum/adapter"
```

Comment:
Let's rename to `ethereumAdapter`

We have the crosschainConnector (the service) which has an adapter
Calling the adapter a connector too is confusing